dotnetv3/dynamodb/FromSQL/UpdateItemDataModelExample/UpdateItemDataModelExample/UpdateItemDataModel.cs

```diff
@@ -158,6 +96,47 @@ static void Main(string[] args)
                 Console.WriteLine("Could not update the order's status.");
             }
         }
+
+        /// <summary>
+        /// Update an item in a DynamoDB table that uses the modified
+        /// data model describe in the Entry class.
+        /// </summary>
+        /// <param name="context">The initialized DynamoDB context used
+        /// to update the item in the DynamoDB table.</param>
+        /// <param name="id">The Id of the item to be updated.</param>
```

Comment:
'Id' is lower case previously, so maybe keep it that way.

Reply:
changed